addons/my-ocular/addon.json

```
@@ -1,15 +1,16 @@
 {
-  "name": "ocular integration",
+  "name": "Ocular integration",
```

`ocular` (all lowercase) is the official name, I'm pretty sure it shouldn't be capitalized.
```suggestion
  "name": "ocular integration",
```

---

But it looks better? @jeffalo what do you think?

---

ocular is spelt ocular.